.github/scripts/update_docker_versions.py

```diff
@@ -81,8 +85,12 @@ def annotate(
                     image, v_current, available = get_quayio_versions(image_details)
                 else:
                     image, v_current, available = get_dockerhub_versions(image_details)
-                stable_versions = [v for v in annotate(available, stable_only=True)]
-                v_latest = sorted(stable_versions, key=lambda v: v[1], reverse=True)[0][0]
+                # Consider only stable versions unless there are none available
+                if not (candidate_versions := [v for v in annotate(available, stable_only=True)]):
```

Comment:
```suggestion
                if not (candidate_versions := annotate(available, stable_only=True)):
```

Isn't this the same?

Reply:
Yes - not sure why it was written this way before.